web/blog/vast-v2.1/index.md

```diff
@@ -8,25 +8,47 @@ tags: [release, performance]
 ---
 
 [VAST v2.1][github-vast-release] is out! This release comes with a particular
-focus on performance, and brings a new utility for optimizing databases in
-production.
+focus on performance and reducing the size of VAST databases, and brings a new
+utility for optimizing databases in production.
```

Comment:
This reads a bit rough to me, with the `and ... and,`. Maybe we can split that into two sentences, potentially circling back in that the tool allows us to take full advantage of these changes and improvements also for existing databases?